src/middleware.ts

```diff
@@ -1,16 +1,24 @@
 import { NextResponse } from "next/server";
 import { NextRequest } from "next/server";
-const baseUrl = "https://t-web-project.vercel.app";
```

Comment:
change back baseUrl to https://t-web-project.vercel.app use localhost when you work localy.